aggregate_spatial.json

```diff
@@ -63,15 +57,15 @@
         }
     ],
     "returns": {
-        "description": "A vector data cube with the computed results. The vector data cube implicitly gets restricted to the bounds of the polygons as if ``filter_polygon()`` would have been used with the same values for the corresponding parameters immediately before this process.\n\nThe computed value is stored in dimension with the name that was specified in the parameter `name`.\n\nThe computation also stores information about the total count of pixels (valid + invalid pixels) and the number of valid pixels (see ``is_valid()``) in each geometry. These values are stored as attributes of the result value with the attribute names `total_count` and `valid_count`.",
+        "description": "A vector data cube with the computed results. The vector data cube implicitly gets restricted to the bounds of the geometries as if ``filter_spatial()`` would have been used with the same values for the corresponding parameters immediately before this process.\n\nThe computed value is stored in dimension with the name that was specified in the parameter `name`.\n\nThe computation also stores information about the total count of pixels (valid + invalid pixels) and the number of valid pixels (see ``is_valid()``) in each geometry. These values are stored as attributes of the result value with the attribute names `total_count` and `valid_count`.",
```

Comment:
If the `total_count` and `valid_count` are stored for each geometry, shouldn't they be values inside the vector cube? Or are these attributes meant to be arrays with the same cardinality as the spatial dimension of the vector cube?

Reply:
@mkadunc Oh, yes, that's a leftover from the time where we returned GeoJSON only. See the most recent commit for the change.